e2e/Cargo.toml

```diff
@@ -8,6 +8,6 @@ crossterm = "0.26.1"
 lazy_static = "1.4.0"
 portpicker = "0.1.1"
 rand = "0.8.5"
-reqwest = { version = "0.11.10", features = ["blocking", "json"] }
+reqwest = { version = "0.11.10", features = ["blocking", "json", "rustls-tls"] }
```

Comment:
I think we need to disable default features here, right?

Reply:
`cargo tree` says otherwise. Reqwest isn't bringing in native-tls or openssl with how it is now.

```
├── reqwest v0.11.18
│   ├── base64 v0.21.1
│   ├── bytes v1.4.0
│   ├── encoding_rs v0.8.32
│   │   └── cfg-if v1.0.0
│   ├── futures-core v0.3.28
│   ├── futures-util v0.3.28 (*)
│   ├── h2 v0.3.19 (*)
│   ├── http v0.2.9 (*)
│   ├── http-body v0.4.5 (*)
│   ├── hyper v0.14.26 (*)
│   ├── hyper-rustls v0.24.0
│   │   ├── http v0.2.9 (*)
│   │   ├── hyper v0.14.26 (*)
│   │   ├── rustls v0.21.1
│   │   │   ├── log v0.4.17 (*)
│   │   │   ├── ring v0.16.20 (*)
│   │   │   ├── rustls-webpki v0.100.1
│   │   │   │   ├── ring v0.16.20 (*)
│   │   │   │   └── untrusted v0.7.1
│   │   │   └── sct v0.7.0 (*)
│   │   ├── rustls-native-certs v0.6.2
│   │   │   ├── openssl-probe v0.1.5
│   │   │   └── rustls-pemfile v1.0.2
│   │   │       └── base64 v0.21.1
│   │   ├── tokio v1.28.1 (*)
│   │   └── tokio-rustls v0.24.0
│   │       ├── rustls v0.21.1 (*)
│   │       └── tokio v1.28.1 (*)
│   ├── ipnet v2.7.2
│   ├── log v0.4.17 (*)
│   ├── mime v0.3.17
│   ├── mime_guess v2.0.4
│   │   ├── mime v0.3.17
│   │   └── unicase v2.6.0
│   │       [build-dependencies]
│   │       └── version_check v0.9.4
│   │   [build-dependencies]
│   │   └── unicase v2.6.0 (*)
│   ├── once_cell v1.17.1
│   ├── percent-encoding v2.2.0
│   ├── pin-project-lite v0.2.9
│   ├── rustls v0.21.1 (*)
│   ├── rustls-pemfile v1.0.2 (*)
│   ├── serde v1.0.163 (*)
│   ├── serde_json v1.0.96 (*)
│   ├── serde_urlencoded v0.7.1 (*)
│   ├── tokio v1.28.1 (*)
│   ├── tokio-rustls v0.24.0 (*)
│   ├── tower-service v0.3.2
│   ├── url v2.3.1 (*)
│   └── webpki-roots v0.22.6 (*)
```

Reply:
Oh, interesting. Is that from running inside the e2e directory?

Reply:
I added `default-features = false` anyway, as it brings in fewer dependencies.

Reply:
e2e is not in the same workspace, so that's why it does not show up in the tree. Regardless, due to not being in the tree, it doesn't really matter whether it uses native-tls.